### PR TITLE
Token yields update

### LINKED ISSUE #17: Add new yield APRs
* sFRAX
* ankrETH
* stakewise osETH
* plsRDNT
* etherfi
* stERN (https://2ch9hbg8hh.execute-api.us-east-1.amazonaws.com/dev/api/vault/0x3eE6107d9C93955acBb3f39871D32B02F82B78AB:0xa) OP

### PR DESCRIPTION
**Pull request by gmbronco**
closes #17

### CODE REVIEW

File: modules/network/arbitrum.ts [hunk 1]
```diff
@@ -175,6 +175,25 @@ const arbitrumNetworkData: NetworkData = {
...
+            },
+            ankrETH: {
+                tokenAddress: '0xe05a08226c49b636acf99c40da8dc6af83ce5bb3',
+                sourceUrl: 'https://api.staking.ankr.com/v1alpha/metrics',
+                path: 'services.{serviceName == "eth"}.apy',
+                isIbYield: true,
+            },
+            plsRDNT: {
+                tokenAddress: '0x6dbf2155b0636cb3fd5359fccefb8a2c02b6cb51',
+                sourceUrl: 'https://plutusdao.io/api/getPlsRdntInfo',
```
Comment (franzns) [inline modules/network/arbitrum.ts]: is the cloudflare captcha fixed?

File: modules/network/fantom.ts [hunk 2]
```diff
@@ -246,6 +246,14 @@ const fantomNetworkData: NetworkData = {
...
             },
         },
+        defaultHandlers: {
+            ankrETH: {
+                tokenAddress: '0x12d8ce035c5de3ce39b1fdd4c1d5a745eaba3b8c',
+                sourceUrl: 'https://api.staking.ankr.com/v1alpha/metrics',
+                path: 'services.{serviceName == "eth"}.apy',
+                isIbYield: true,
+            },
+        },
```
Comment (franzns) [inline modules/network/fantom.ts]: this is already added via the ankr YB apr service. remove the ankr specific service if it works with the default one. For all chains

**Review by franzns — CHANGES_REQUESTED** [review]

File: modules/network/fantom.ts (hunk 2 above)
Comment (gmbronco) [reply modules/network/fantom.ts]: good point, missed that, removed the custom one.

File: modules/network/arbitrum.ts (hunk 1 above)
Comment (franzns) [reply modules/network/arbitrum.ts]: Shall we remove this until we receive confirmation it's working?

Comment (gmbronco) [reply modules/network/arbitrum.ts]: it's ready to go

**Review by franzns — APPROVED** [review]

**Pull request merged**